### DISCUSSION TITLE
DistroAV & NDI™ Runtime & Tools : Quick Install guide

### DISCUSSION THREAD

**Discussion by Trouffman** [DistroAV/DistroAV#831] [Show and tell]
Please refer to the latest version of the installation process on the Wiki : https://github.com/DistroAV/DistroAV/wiki/1.-Installation

The discussion below is kept for history and discussion in the future.
========

Install DistroAV on windows with a single Command line:

`winget install --exact --id DistroAV.DistroAV`

========

========

Ever wanted to install the NDI™ Runtime quickly without wasting time on Windows?
Use the built-in Windows Package Manager and  you can get them in one command line.

Open a command line (Start menu > type CMD)
`winget install --exact --silent --id NDI.NDIRuntime`

Same process for the NDI Tools (latest)
`winget install --exact --silent --id NDI.NDITools` 

K. thx. bye.

### NDI™ 6 Runtime:
- Windows : http://ndi.link/NDIRedistV6
- MacOS: http://ndi.link/NDIRedistV6Apple
- Linux: https://downloads.ndi.tv/SDK/NDI_SDK_Linux/Install_NDI_SDK_v6_Linux.tar.gz
Linux install script: [./CI/libndi-get.sh](https://github.com/obs-ndi/obs-ndi/blob/master/CI/libndi-get.sh)

### NDI™ 6 Tools:
- Windows : https://downloads.ndi.tv/Tools/NDI%206%20Tools.exe
- MacOS: https://downloads.ndi.tv/Tools/NDIToolsInstaller.pkg
- Linux: Not released

### Known issue and common errors

=======================

**Comment by paulpv** [comment]
Cool to know!

**Comment by Trouffman** [comment]
I have added the manual Download link as well. Will use this post as a "reference" whenever someone needs help about it.

**Reply by iq0099** [reply]
![image](https://github.com/user-attachments/assets/c1951838-541e-4020-8bf3-63c97834a821)

**Reply by iq0099** [reply]
I simply downloaded the NDI - OBS plug-in, and installed it. The install fails immediately!

**Reply by paulpv** [reply]
@iq0099 What is the server name or address?

**Comment by paulpv** [comment]
FYI, the links are still not working (apparently due to them "moving"), even after reporting the issue to NewTek forums and getting a response that they are working on it. :/

**Comment by Trouffman** [comment]
New version just released today : 

Tools 
Windows : https://downloads.ndi.tv/Tools/NDI%205%20Tools.exe
MacOS : https://downloads.ndi.tv/Tools/NDIToolsInstaller.pkg

Runtime : 
- still no new direct link yet.

**Comment by Trouffman** [comment]
And The new link for the NDI Runtimw have been added & tested!

Note that the one-liner will soon change to remove the Newtek Mention and only be with NDI branding. Will update this task when ready.

**Reply by paulpv** [reply]
Can you explain that last part better?

**Reply by paulpv** [reply]
Oh, by "one-liner", you mean the `winget` commands?  
Understood.

I will also post the Mac and Linux command-line ways to get this too.

**Comment by pianobelloso** [comment]
The MacOS Runtime link is not downloading latest version (5.5.3). I am only able to use successfully in OBS with that version, NOT the V5 version linked above! The [Releases page](https://github.com/obs-ndi/obs-ndi/releases/tag/4.11.1) has the updated version btw. Just not this master page.

**Comment by Trouffman** [comment]
Updated the command lines in the Opening Post.
Seems there was a renaming of the Winget package.

**Comment by NineTaleFox** [comment]
So, while I'm not great at cmd.exe and such I can't get the command prompt or the NDI 5 runtime link to load.... and there doesn't seem to be a working source. Not for nothing tools ... don't meet my needs ... as a nice way of putting my opinion on them. Any thoughts from people better at this than myself?

**Comment by sebastian-beca** [comment]
anybody have a working ink to NDI 5 Runtime for MacOS?

**Reply by paulpv** [reply]
https://downloads.ndi.tv/Tools/NDIToolsInstaller.pkg (from above) downloads fine for me

**Reply by pianobelloso** [reply]
Does that installer include Runtime? I thought it was separate.

**Reply by sebastian-beca** [reply]
I've installed the NDI Tools and it doesn't seem to have the runtime as when I start OBS I get this message:
<img width="397" alt="image" src="https://github.com/obs-ndi/obs-ndi/assets/3092116/e5c9758a-fc07-412d-b3d2-8be7b209f834">

**Reply by paulpv** [reply]
A little odd that MacOS NDI Tools does not install the NDI runtime.
I guess that could/may be the case.

Regardless, does their shown URL of http://ndi.link/NDIRedistV5Apple work for you?
These URLs can also be found in the code at:
https://github.com/obs-ndi/obs-ndi/blob/master/lib/ndi/Processing.NDI.Lib.h#L53-L64
(lines 53, 57, and 64)

**Reply by sebastian-beca** [reply]
it seems to be working again!

**Reply by Pawedcats** [reply]
This is an old issue in OBS which apparently has still not been resolved. You can see a lot of people trying to figure it out here, with a supposed solution that didn't work for me:  
https://obsproject.com/forum/threads/obs-ndi-newtek-ndi%E2%84%A2-integration-into-obs-studio.69240/page-41#post-595250

And also a supposed solution listed at the bottom of the page here:
https://obsproject.com/forum/threads/ndi%E2%84%A2-runtime-not-found-macos-ventura.161798/

**Comment by rokel-race** [comment]
Looks like the Windows package has been renamed from `Newtek` to `NewTek`, hence command is now:

```cmd
winget install --exact --silent --id NewTek.NDI5Runtime
```

**Comment by Trouffman** [comment]
NDI6 is out today, this might need some days before it is properly deployed to the new winget repo. I will also look to get the names better streamlined as NDI is not linked to newtek and NDI5 is outdated (as of today as well).

**Comment by Trouffman** [comment]
The new ID for the winget command is published for NDI Tools :
`winget install NDI.NDITools`

**Comment by Trouffman** [comment]
Updated with the new Winget command for NDI 6 and future versions.

On windows 10/11 runtime are a drop-in replacement.

**Comment by theking2** [comment]
Is there choco install for the ndi-runtime? there is one for obs-ndi

**Reply by Trouffman** [reply]
There is no officiel NDI runtime distribution by NDI on chocolatey

**Comment by nuthinfancy** [comment]
Doesn't OBS put out a pre release so y'all can check if you need to update stuff?
Feels like Windows 3.11

**Reply by paulpv** [reply]
I don't understand your comment.
What does OBS putting out a pre-release have to do with us?
We work fine on any version of OBS >= v30 ~99% of the time, especially if you follow our official instructions at https://github.com/DistroAV/DistroAV/wiki/1.-Installation

**Reply by RayneYoruka** [reply]
Seems to be a firewall issue as per usual. Please make sure to follow https://github.com/DistroAV/DistroAV/wiki/2.-Troubleshooting and to report on the right places

**Reply by nuthinfancy** [reply]
Ooops. Forgot to add cam to NDI Tools Access Manager. Working now
Deleted previous comment so as not to confuse.

**Comment by gdhgdhgdh** [comment]
Admin edit : this is NOT a recommended solution. Does not apply anymore.

With the direct download of the MacOS Runtime seemingly dead, I installed the Tools, and then searched the whole filesystem for the correct library file. I found it lurking inside one of the tools, so I copied it into `/usr/local/lib` and OBS with the NDI plugin started up perfectly :)

```
sudo cp /Library/CoreMediaIO/Plug-Ins/DAL/NDIVideoOut.plugin/Contents/Frameworks/libndi.dylib /usr/local/lib/
```

**Comment by MrK-VN** [comment]
<img width="844" alt="Screenshot 2025-06-13 at 09 52 25" src="https://github.com/user-attachments/assets/0b973e96-e5fd-4462-824b-bd88b0fccf60" />
I’ve already installed the NDI Runtime, but when I open OBS, it still shows an error. What should I do next?

**Reply by RayneYoruka** [reply]
Deprecated and not supported plugin version. [Read the wiki with the uninstall instructions. ](https://github.com/DistroAV/DistroAV/wiki/1.-Installation#uninstall)

**Comment by datamattsson** [comment]
Decently new libNDI 6.2.1 for Mac: `NDI SDK APPLE 10:26:45 Aug 21 2025 6.2.1.0`

**Comment by deseipel** [comment]
FYI -  the cert from NewTek expired on 3/5:   
<img width="818" height="488" alt="image" src="https://github.com/user-attachments/assets/ce9203d4-65a1-4b90-a98b-c4ed7e5fb5e3" />

So NDI Runtime will not install via command line.  You'd have to manually dl it and prob unblock it.

**Reply by RayneYoruka** [reply]
Thanks for the headsup, I'll pass it along asap.

**Comment by BitRate27** [comment]
The certificate is still considered valid and Windows will allow the current NDI Runtime install executable to be run forever. This is because the Signing date timestamp is set (February 17, 2026) which is in the validity range of the certificate. NDI will need to get a new certificate if it wants to sign any new executables. They should find this out when they go to sign it. 

Chrome will not let you download the "http://ndi.link/NDIRedistV6" link in the Wiki, because it is not secured. I changed the Wiki to be "https://ndi.link/NDIRedistV6" and it now downloads. I can also run the downloaded file without unblocking it as the certificate is valid.

**Comment by Trouffman** [comment]
All the NDI links that used to be http:// only are now https://
